marketplace/commercelayer-sku/package.json

```diff
@@ -0,0 +1,38 @@
+{
+  "name": "commercelayer-sku",
+  "version": "0.1.0",
+  "private": true,
+  "devDependencies": {
+    "@babel/core": "7.3.4",
+    "@babel/plugin-proposal-class-properties": "7.3.4",
+    "@babel/plugin-transform-runtime": "7.3.4",
+    "@babel/preset-env": "7.3.4",
+    "@babel/preset-react": "7.0.0",
+    "@contentful/contentful-extension-scripts": "0.12.0",
+    "cssnano": "4.1.10",
```

Comment:
Is this used? I can't see where.

Reply:
I removed the library, tested it and things behaved and looked as expected so committed a version without the library.